tox.ini

```diff
@@ -1,41 +1,45 @@
 [tox]
 envlist =
-    {py3.5,py3.6,py3.7,py3.8}-{sqlite,postgresql}-trytond-{5.0,5.2,5.4,5.6,latest}
+    {py3.5,py3.6,py3.7,py3.8,py3.9}-{sqlite,postgresql}-trytond-{5.0,5.2,5.4,5.6,latest}
```

Comment:
The trytond dependency matrix makes me wonder: is a single version of this module compatible with all those trytond versions? Isn't this module gonna follow the conventional trytond-module branch structure (one branch per series, cherrypicking and backporting developments from the latest, each one constraining its compatible trytond version in setup.py)?

I think this tox-dependency-matrix approach is intended for generic libraries that can work with many trytond series like factory_trytond. But if trytoncalidae-authentication_dummy==X.Y.Z already constraints to trytond>=X.Y<X.Y+1 then all this is rather confusing!